dpt/Cargo.toml

```diff
@@ -8,18 +8,18 @@ repository = "https://github.com/sorpaas/rust-devp2p"
 edition = "2018"
 
 [dependencies]
-secp256k1-plus = "0.5"
-rand = "0.3.12"
-sha3 = "0.6"
 etcommon-bigint = "0.2"
 etcommon-hexutil = "0.2"
 etcommon-rlp = "0.2"
```

Comment:
Those should be replaced by `primitive-types`, `rlp` in the future. Not sure if there's a new crate for `hexutil`.

For many cases, things may also be available in `ethereum-` namespace.